config/manager/manager.yaml

```diff
@@ -2,25 +2,25 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
-  name: system
+    control-plane: ssp-operator
```

Comment:
I think we should not add this label to the `kubevirt` namespace. At least until we know how is this label used. I will try to find out.

Reply:
Right, good point

Reply:
Removed the whole label